README.md

```diff
@@ -1,2 +1,68 @@
 # http-client
 An opinionated, isomorphic HTTP client.
+
+### Usage
+
+#### Import httpClient
+```js
+import {httpClient} from '@digitalbazaar/http-client';
+```
+
+#### GET a JSON response in the browser
+```js
+try {
+  result = await httpClient.get('http://httpbin.org/json');
+  return result.data;
+} catch(e) {
+  // status is HTTP status code
+  // data is JSON error from the server
```

Comment:
A bit tricky to follow the error path, but if you ask for json, and got back a 404 with html content, data would just be undefined right?  Maybe this should say "data is JSON error from the server if available"?